deploy/go-carbon.conf

```diff
@@ -238,6 +238,12 @@ scan-frequency = "5m0s"
 #  memory usage (by setting both trie-index and trigram-index to true).
 trie-index = false
 
+# This provides the ability to query for new metrics without any wsp files
+# i.e query for metrics present only in cache. Does a cache-scan and
+# populates index with metrics with or without corresponding wsp files.
+# Disabled by default.
+cache-scan = false
```

Comment:
If my comment on extra memory overhead is correct, could you mention the side effect of keeping a copy of all the files/dirs in memory here?

Reply:
Sure. I will update the comment and mention about the memory overhead.